### DISCUSSION TITLE
77 tokens limit

### DISCUSSION THREAD

**Discussion by amida168** [azuritecoin/OnnxDiffusersUI#29] [General]
When I use Automatic1111, I can use prompt for more than 77 tokens. However, the end of the prompt is just ignored if it's over 77 tokens with OnnxDiffusersUI. Is this the limit imposed by the diffusers?

**Comment by azuritecoin** [comment]
Yeah it's a limitation. There's a community pipeline that has enabled longer and wheighted prompts. [See here](https://github.com/huggingface/diffusers/tree/main/examples/community#long-prompt-weighting-stable-diffusion)

**Comment by Tjololo12** [comment]
I've been unable to get a community pipeline working with the onnxUI. Where's the 77 token limit implemented, so I can see what needs to potentially be updated to remove it?

**Reply by azuritecoin** [reply]
well good news. the LPW bug has been fixed. I'll try to update my UI.

**Reply by Tjololo12** [reply]
Oh awesome, I appreciate that! I was trying to see if I could figure it out to make a PR and save you the work, but unfortunately I couldn't figure it out. Thanks for updating it!

**Reply by azuritecoin** [reply]
It's been updated

**Reply by Tjololo12** [reply]
Excellent! I'm now seeing "Prompt was truncated. Try to shorten the prompt or increase max_embeddings_multiples"...is that something that I can increase?

**Reply by azuritecoin** [reply]
How long is your prompt? lol

You can increase the max_embeddings_multiples on line 365 in `lpw_pipe.py`. Make sure you restart the onnxUI afterwards.

**Reply by Tjololo12** [reply]
Haha thanks, that worked perfectly. I was copy-pasting sections of academic papers just to see what it would come up with. I appreciate how responsive you've been!
